node/tests/SharedTests.ts

```diff
@@ -316,6 +316,39 @@ export function runBaseTests<Context>(config: {
         config.timeout,
     );
 
+    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+        `msetnx test_%p`,
+        async (protocol) => {
+            await runTest(async (client: BaseClient) => {
+                const key1 = "{key}-1" + uuidv4();
+                const key2 = "{key}-2" + uuidv4();
+                const key3 = "{key}-3" + uuidv4();
+                const nonExistingKey = uuidv4();
+                const value = uuidv4();
+                const keyValueMap1 = {
+                    [key1]: value,
```

Comment:
Why is the key wrapped in `[]`?

Reply:
i used the same formatting as the MSET command

Reply:
That is the valid syntax to use variable value